fairscale/experimental/nn/mevo.py

```diff
@@ -184,15 +183,21 @@ def backward(ctx: Any, *args: Any) -> Any:
         i, w = ctx.saved_tensors
         assert i.requires_grad
         assert w.requires_grad
+        # We use ``detach()'' to ensure the backward call below does not
```

Comment:
it really helps, I never realized that custom backward functions could themselves build a new autograd graph, I always assumed (I was wrong) that autograd was hands off there.. thanks for the comments @min-xu-ai !